data_destributor.gemspec

```diff
@@ -14,5 +14,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rake"
   s.add_development_dependency 'vcr'
   s.add_development_dependency 'webmock'
-
+  s.add_development_dependency 'timecop'
```

Comment:
Mindre ting med hold runtime_dependencies og development_dependencies i to grupper adskildt med en tom linje og internt sorteret alfabetisk, samt brug af " i stedet for '. 
Så bliver jeg glad :-)

Reply:
Done